android/app/src/main/java/network/mysterium/ui/ProposalsViewModel.kt

```diff
@@ -313,6 +313,10 @@ class ProposalsViewModel(private val sharedViewModel: SharedViewModel, private v
                     .filter { it.countryCode != "" }
                     .map { ProposalViewItem.parse(it, favoriteProposals) }
 
+            if (applyFilter(filter, allProposals).size == 0) {
```

Comment:
I'm thinking there could be a legit case where no proposals match the filters. On desktop, I'm fetching proposals and quality separately, and proposals with quality `unknown` are shown by default. Do you think we could do the same on mobile?

Reply:
Changed to show `unknown` quality by default.